src/main/java/com/metamx/metrics/JvmThreadsMonitor.java

```diff
@@ -0,0 +1,58 @@
+package com.metamx.metrics;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableMap;
+import com.metamx.emitter.service.ServiceEmitter;
+import com.metamx.emitter.service.ServiceMetricEvent;
+import java.util.Map;
+import org.gridkit.lab.jvm.perfdata.JStatData;
+import org.gridkit.lab.jvm.perfdata.JStatData.LongCounter;
+
+public class JvmThreadsMonitor extends AbstractMonitor
```

Comment:
Why this information is collected via jStat, rather than `ThreadMXBean`? It seems that all those counters are accessible via `ThreadMXBean`.

I know your argument was to "use one tool for everything, instead of two tools". My counter-arguments would be:
 - We anyway still use JMX in other places
 - JMX is standard, jStat is not standard, especially string mnemonics of counters.
 - ThreadMXBean is easier to access, requires less setup code (the whole setup is actually just calling `ManagementFactory.getPlatformMXBeans(ThreadMXBean.class)`, then calling plain methods on the returned `ThreadMXBean`.
 - It's possible to `resetPeakThreadCount()` via `ThreadMXBean`, not sure it is possible via jStat (see below).